### PR TITLE
Add synthetic data config and train bash files

### PR DESCRIPTION
**Pull request by liketheflower**
The synthetic data is generated based on the README and saved in the following path
```
/raid/home/public/JIMMY/SubGNN_data/
```
The synthetic data can be trained by running the following script.
```
./scripts/run_train_config.sh 
```
On DGX, if you want to run the code, pls change the config.py
```
PROJECT_ROOT = Path("/raid/home/public/JIMMY/SubGNN_data/")
```
In this PR:

- the configure file for the generated synthetic dataset is created.
- the shell script is created to run the training code based on the synthetic dataset.

### CODE REVIEW

**Comment by thomasly** [comment]
Thanks, Jimmy. Please use /raid/home/public/YangLiu/SubGNN_data/ as the data root directory so we can make sure the data we will use in the future are the same. Let me know if you have issues accessing the directory.

**Comment by liketheflower** [comment]
> Thanks, Jimmy. Please use /raid/home/public/YangLiu/SubGNN_data/ as the data root directory so we can make sure the data we will use in the future are the same. Let me know if you have issues accessing the directory.

Thanks. Put it in a separate folder to avoid overwriting. Just realized that there is no SubGNN_data folder under "/raid/home/public/YangLiu/". Moved generated synthetic data to "/raid/home/public/YangLiu/", so you can try to play with training the synthetic dataset without change PROJECT_ROOT

The synthetic data is here now:
```
/raid/home/public/yangliu/SubGNN_data
```

**Review by thomasly — CHANGES_REQUESTED** [review]
> > Thanks, Jimmy. Please use /raid/home/public/YangLiu/SubGNN_data/ as the data root directory so we can make sure the data we will use in the future are the same. Let me know if you have issues accessing the directory.
> 
> Thanks. Put it in a separate folder to avoid overwriting. Just realized that there is no SubGNN_data folder under "/raid/home/public/YangLiu/". Moved generated synthetic data to "/raid/home/public/YangLiu/", so you can try to play with training the synthetic dataset without change PROJECT_ROOT
> 
> The synthetic data is here now:
> 
> ```
> /raid/home/public/yangliu/SubGNN_data
> ```

Hi Jimmy,

There is a SubGNN_data folder at:
/raid/home/public/YangLiu/SubGNN_data
Please put your data there. Please completely remove the /raid/home/public/yangliu folder (notice the case) to which I don't even have access.

File: SubGNN/config_files/density/density_config.json [hunk 1]
```diff
@@ -0,0 +1,123 @@
+{
+    "data": {
+        "task":"density"
+    },
+    "no_gpu":true, //add this line to use CPU
```
[the remark this answers is not in the document]
Comment (liketheflower) [reply SubGNN/config_files/density/density_config.json]: Fixed. PTAL :)

**Comment by liketheflower** [comment]
> > Thanks, Jimmy. Please use /raid/home/public/YangLiu/SubGNN_data/ as the data root directory so we can make sure the data we will use in the future are the same. Let me know if you have issues accessing the directory.
> 
> Thanks. Put it in a separate folder to avoid overwriting. Just realized that there is no SubGNN_data folder under "/raid/home/public/YangLiu/". Moved generated synthetic data to "/raid/home/public/YangLiu/", so you can try to play with training the synthetic dataset without change PROJECT_ROOT
> 
> The synthetic data is here now:
> 
> ```
> /raid/home/public/yangliu/SubGNN_data
> ```

I don't have write permission for this folder:
"/raid/home/public/YangLiu/"
And I can not add read and write permission to you as we are not in the same group :(
drwxrwxrwx  4 jimmyshen jimmyshen       4096 Mar  4 22:53 SubGNN_data
drwxr-xr-x 10 yangliu   faculty         4096 Feb 22 10:57 YangLiu

**Comment by liketheflower** [comment]
> > > Thanks, Jimmy. Please use /raid/home/public/YangLiu/SubGNN_data/ as the data root directory so we can make sure the data we will use in the future are the same. Let me know if you have issues accessing the directory.
> > 
> > 
> > Thanks. Put it in a separate folder to avoid overwriting. Just realized that there is no SubGNN_data folder under "/raid/home/public/YangLiu/". Moved generated synthetic data to "/raid/home/public/YangLiu/", so you can try to play with training the synthetic dataset without change PROJECT_ROOT
> > The synthetic data is here now:
> > ```
> > /raid/home/public/yangliu/SubGNN_data
> > ```
> 
> I don't have write permission for this folder:
> "/raid/home/public/YangLiu/"
> And I can not add read and write permission to you as we are not in the same group :(
> drwxrwxrwx 4 jimmyshen jimmyshen 4096 Mar 4 22:53 SubGNN_data
> drwxr-xr-x 10 yangliu faculty 4096 Feb 22 10:57 YangLiu

Sent an email to Shuo to see whether he can help me change the group.

**Comment by liketheflower** [comment]
> > > > Thanks, Jimmy. Please use /raid/home/public/YangLiu/SubGNN_data/ as the data root directory so we can make sure the data we will use in the future are the same. Let me know if you have issues accessing the directory.
> > > 
> > > 
> > > Thanks. Put it in a separate folder to avoid overwriting. Just realized that there is no SubGNN_data folder under "/raid/home/public/YangLiu/". Moved generated synthetic data to "/raid/home/public/YangLiu/", so you can try to play with training the synthetic dataset without change PROJECT_ROOT
> > > The synthetic data is here now:
> > > ```
> > > /raid/home/public/yangliu/SubGNN_data
> > > ```
> > 
> > 
> > I don't have write permission for this folder:
> > "/raid/home/public/YangLiu/"
> > And I can not add read and write permission to you as we are not in the same group :(
> > drwxrwxrwx 4 jimmyshen jimmyshen 4096 Mar 4 22:53 SubGNN_data
> > drwxr-xr-x 10 yangliu faculty 4096 Feb 22 10:57 YangLiu
> 
> Sent an email to Shuo to see whether he can help me change the group.

Two ways to get the data
1) I put the data in the following location
```
/raid/home/public/SubGNN_data
```
and give you the write permission. You can copy it to /raid/home/public/YangLiu/SubGNN_data/ 
2) If you add "others" the write permission (we do not share group :( ), I can move the data to  /raid/home/public/YangLiu/SubGNN_data/

**Pull request merged**